simapp/go.mod

```diff
@@ -23,17 +23,16 @@ require (
 	// this version is not used as it is always replaced by the latest Cosmos SDK version
 	github.com/cosmos/cosmos-sdk v0.50.2
```

Comment:
You don't need your fork's changes here?

Reply:
Done https://github.com/0xPolygon/cosmos-sdk/pull/16/commits/1554abab5ff582a27ef21c3d61f33e8494f98801

Reply:
Oh, I see! I had missed that